queries.sql

```diff
@@ -134,3 +134,89 @@ COUNT(animals.owner_id)
 FROM animals
 INNER JOIN owners ON owners.id = animals.owner_id
 GROUP BY owners.full_name;
+
+/*Vet clinic database: add "join table" for visits*/
+-- Who was the last animal seen by William Tatcher?
+SELECT a.name, v.visit_date
+FROM animals a
+  JOIN visits v ON a.id = v.animal_id
+  JOIN vets ve ON v.vet_id = ve.id
+WHERE ve.name = 'William Tatcher'
+ORDER BY v.visit_date DESC
+LIMIT 1;
+
+-- How many different animals did Stephanie Mendez see?
+SELECT COUNT(*)
+FROM (
+        SELECT v.animal_id from visits v
+            JOIN vets ve ON v.vet_id = ve.id
+        WHERE ve.name = 'Stephanie Mendez'
+        GROUP BY v.animal_id
+    )as xx;
+
+-- List all vets and their specialties, including vets with no specialties.
+SELECT ve.name, s.name
+FROM vets ve
+  LEFT JOIN specializations sp ON ve.id = sp.vet_id
+  LEFT JOIN species s ON sp.species_id = s.id;
+
+  -- List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
+  SELECT a.name
+FROM animals a
+    JOIN visits v ON a.id = v.animal_id
```

Comment:
- [ ] I recommend using 
```
SELECT animals.name, vistis.visit_date
  FROM visits
  JOIN animals ON animals.id = visits.animal_id
```
It will take the name of animals and date of visits from the visits table and join it with animals and vets. (`JOIN vets as ve ON v.vet_id = ve.id` is OK) 😉

Reply:
Thanks for the suggestion, I got the same result but also with the visit date.  I already included it into the queries.sql
![image](https://user-images.githubusercontent.com/109246702/221238437-ef04c3fb-37a9-4ad2-86a2-800c548d54ad.png)